### ISSUE TITLE
NTR: electrolyte imbalance

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#5100]
**Preferred term label:**
electrolyte imbalance

**Synonyms**
water-electrolyte imbalance

**Definition (free text, please give PubMed ID)**
An abnormality in the concentration of electrolytes in the body (e.g., in the urine, blood or tissue.)
PMID:25215103

**Parent term (use hpo.jax.org/app)**
HP_0012337 'Abnormal homeostasis'

or would this be a better parent?
HP_0011032 'Abnormality of fluid regulation'

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
MONDO_0100094 Cannabinoid hyperemesis syndrome

Your nano-attribution (ORCID)
https://orcid.org/0000-0001-5208-3432

See monarch-initiative/mondo#1084

**Comment by pnrobinson** [comment]
I actually do not think this is a great term, because it is an inference (things are out of balance) rather than an observation ("low blood sodium").

**Comment by nicolevasilevsky** [comment]
No problem, let's close.
